src/mood.ts

```diff
@@ -73,8 +79,23 @@ export function meatMood(urKels = false, embezzlers = false): Mood {
   mood.skill($skill`The Spirit of Taking`);
   mood.skill($skill`Drescher's Annoying Noise`);
   mood.skill($skill`Pride of the Puffin`);
-  if (myClass() !== $class`Pastamancer`) mood.skill($skill`Bind Lasagmbie`);
-
+  if (myClass() !== $class`Pastamancer`) {
+    const isWorthIt =
+      (baseMeat * getModifier("Meat Drop", $effect`Pasta Eyeball`)) /
+        100 /
+        mpCost($skill`Bind Lasagmbie`) >
+      Math.max(
+        (1.5 * myLevel() + 5) / 80,
+        (2.5 * myLevel()) / mallPrice($item`generic mana potion`)
+      );
```

Comment:
can you break this into sub-computations with names? A little incomprehensible right now.

Reply:
sure!